build.sbt

```diff
@@ -20,6 +21,18 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
     libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.3.0"
   )
 
+lazy val testKit = crossProject(JSPlatform, JVMPlatform)
+  .crossType(CrossType.Pure)
+  .in(file("modules/testkit"))
+  .settings(commonSettings)
+  .settings(
+    name := "cats-time-testkit",
+    libraryDependencies ++= Seq(
+      "org.typelevel"          %%% "cats-testkit-scalatest"  % "2.1.5",
```

Comment:
Are you sure we need these dependencies? the testKit module should only export arbitraries

Reply:
Yeah, we need at least `scalacheck` dependency since `org.scalacheck.{Arbitrary, Gen}` is being used in `TimeArbitraries`.
Also, we need a `scala-collection-compat` for cross-build with scala `2.12`.

Reply:
One improvement I can think of is that we can extract dependencies into separate `val` so that we will not repeat them here. What do you think?

Reply:
I'd rather depend directly on `scalacheck`. There maybe downstream projects that want to depend only on the arbitraries and not bring `cats-testkit-scalatest` along. Specially for projects that don't use `scalatest`

Reply:
Make sense, let's stick to `scalacheck` only. 
Fixed.